Applications/Games/Lego Rock Raiders/Local/script.json

```diff
@@ -0,0 +1,14 @@
+{
+    "scriptName"                 : "Local",
+    "id"                         : "lego_rock_raiders",
+    "compatibleOperatingSystems" : [
+        "MACOSX",
+        "LINUX"
+    ],
+    "testingOperatingSystems" : [
+        "MACOSX",
```

Comment:
Is it still correct that this is testing?

Reply:
Yes.